index.html

```diff
@@ -260,8 +260,59 @@ <h2>About me</h2>
           </ul> 
         </div> 
       </div>
+    </div>
+  </section>
 
+  <section class="contact-form" id="contact">
+    <div class="contact">
+      <h2 class="form-title">I'm always<br>interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+      <form class="form" id="form" action="https://formspree.io/f/xrgdolkb" method="POST">
+        <ul>
+          <li>
+            <input class="form-input" type="text" name="name" placeholder="Full Name" maxlength="30" required>
+          </li>
+          <li>
+            <input id="email" class="form-input" type="email" name="email" placeholder="Email address" required>
+          </li>
+          <li>
+            <textarea class="textmsg" name="message" maxlength="500">Enter text here</textarea required>
```

Comment:
- [ ] [Required] Good job so far! It would be semantic and nice if you paste `"Enter text here"` text and `required` property  inside of the opening tag like a:
-  `<textarea class="textmsg" name="message" maxlength="500" placeholder=" Enter text here" required></textarea>` 👍🏼